db/migrate/20220614163343_change_name_on_users.rb

```diff
@@ -0,0 +1,5 @@
+class ChangeNameOnUsers < ActiveRecord::Migration[6.0]
+  def change
+    change_column :users, :name, :string, null: false
+  end
+end
```

Comment:
Adding a `null constraint` to an existing column , like in this migration, may make troubles in the future. 
Here's nicely explained why: https://testsuite.io/manage-database-null-contraints
That's why Rails offers the `change_column_null`-migration (see the link to the guides in my previous comment). 

So, consider to practice the column_null migration in this project. Your call, of course.